doc/rst/feat_adat_tx.rst

```diff
@@ -0,0 +1,134 @@
+ADAT Transmit
+=============
+
+``lib_xua`` supports the development of devices with ADAT transmit functionality through the use of
+`lib_adat <https://github.com/xmos/lib_adat>`__. The XMOS ADAT transmitter runs on a single core and supports transmitting 8 channels of
```

Comment:
We're just using "thread" instead of "core" now.

Reply:
I'm going to find and replace all references to core with thread :)

Reply:
careful you don't catch xcore! :p